docs/config.md

```diff
@@ -4,10 +4,10 @@
 
 ### type
 
-- 类型：`'login' | 'token' | 'naifu' | 'sd-webui'`
+- 类型：`'login' | 'token' | 'naifu' | 'sd-webui' | 'stable-horde'`
 - 默认值：`'token'`
 
-登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu` 和 `sd-webui` 对应着其他类型的后端。
+登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu`、 `sd-webui` 和 `stable-horde` 对应着其他类型的后端。
```

Comment:
```suggestion
登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu`、`sd-webui` 和 `stable-horde` 对应着其他类型的后端。
```

pangu

Reply:
Oh, so the dunhao symbol is not panguable.